wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WVideo.java

```diff
@@ -218,7 +220,9 @@ public String getMediaGroup() {
 	 * @param mediaGroup The media group name.
 	 */
 	public void setMediaGroup(final String mediaGroup) {
-		getOrCreateComponentModel().mediaGroup = mediaGroup;
+		if (getMediaGroup() == null || !getMediaGroup().equals(mediaGroup)) {
```

Comment:
dopn't need a new component model if getMediaGroup() == null and mediaGroup == null.